manual/mapper/config/README.md

```diff
@@ -175,3 +175,82 @@ dependencies {
 
 You'll also need to install a Lombok plugin in your IDE (for IntelliJ IDEA, [this
 one](https://plugins.jetbrains.com/plugin/6317-lombok) is available in the marketplace).
+
+#### Kotlin
+
+[Kotlin](https://kotlinlang.org/) is an alternative language for the JVM. Its compact syntax and
+native support for annotation processing make it a good fit for the mapper.
+
+To set up your project, refer to the Kotlin website:
+
+* Maven: configure [dual compilation][maven_kotlin_java] of Kotlin and Java sources. In addition,
+  you'll need an additional execution of the [kotlin-maven-plugin:kapt][maven_kapt] goal with the
+  mapper processor before compilation:
```

Comment:
Would be nice to clarify that this execution belongs under `kotlin-maven-plugin`.